python/lsst/analysis/ap/zooniverseCutouts.py

```diff
@@ -397,6 +397,11 @@ def _annotate_image(fig, source, flags):
     if any(flags[flags_shape]):
         fig.text(0.87, 0.83, "SHAPE", color="red", fontweight="bold")
 
+    # rb score
+    fig.text(0.73, 0.79, f"RB:{source['spuriousness']:.03f}",
+             color="red" if source['spuriousness'] < 0.5 else "green",
```

Comment:
I'm tempted to suggest using something like the matplotlib `cool` or `cool_r` colormap, with the spuriousness (or `log(spuriousness)`?) to color it, instead of making a hard cutoff. But that might not actually be much of a gain.